packages/terra-consumer-nav/src/Nav.jsx

```diff
@@ -17,6 +16,44 @@ const propTypes = {
   // expand object shape
   quickLinks: PropTypes.arrayOf(PropTypes.object),
   navItems: PropTypes.arrayOf(PropTypes.object),
+  helpItems: PropTypes.arrayOf(PropTypes.shape({
+    slug: PropTypes.string,
+    nav_type: PropTypes.oneOf(['GROUPING', 'EXTERNAL', 'APPLICATION', 'MODAL']),
+    target: PropTypes.string,
+    text: PropTypes.string.isRequired,
+    uri: PropTypes.string.isRequired,
+    icon: PropTypes.oneOfType([
+      PropTypes.string,
+      PropTypes.element,
+    ]),
+    children: PropTypes.arrayOf(PropTypes.shape({
+      slug: PropTypes.string,
+      nav_type: PropTypes.oneOf(['GROUPING', 'EXTERNAL', 'APPLICATION', 'MODAL']),
+      text: PropTypes.string.isRequired,
+      uri: PropTypes.string.isRequired,
+      icon: PropTypes.oneOfType([
+        PropTypes.string,
+        PropTypes.element,
+      ]),
+      children: PropTypes.array,
```

Comment:
There was a discussion and Mike told to ignore third level nav items. Please check this draft object nav is expecting and help item has children of children.
https://jira2.cerner.com/browse/PORTALDEV-34187